extensions/mssql/package.json

```diff
@@ -419,6 +419,29 @@
     "connectionProvider": {
       "providerId": "MSSQL",
       "displayName": "Microsoft SQL Server",
+      "iconPath": [
+        {
+          "condition": "serverInfo.isCloud",
```

Comment:
This should be `"when"`, not `condition`. That is the syntax for any condition in json contributions across VSCode and Azure Data Studio

Reply:
fixed.

Reply:
Also are you sure that serverInfo.isCloud actually works as intended? I was trying to use that for my menu contributions and it was taking the entire name as a key instead of parsing the . as a property of the top level object.